src/index.js

```diff
@@ -4,11 +4,14 @@ import './index.css';
 import App from './App';
 import GlobalStyles from './GlobalStyles'
 import * as serviceWorker from './serviceWorker';
+import { BrowserRouter as Router } from 'react-router-dom'
 
 ReactDOM.render(
   <React.StrictMode>
     <GlobalStyles />
+    <Router>
     <App />
+    </Router>
```

Comment:
Ahh okay da ist der Router, bitte vergiss den oberen Kommentar!

Reply:
Wenn du den Router aber nur in der App.js hast, sparst du dir hier den Import

Reply:
Hab ich geändert, danke!